src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(value -> value.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::valueOf)
+                .filter(num -> num % 2 == 0)
+                .min(Integer::compare)
```

Comment:
```suggestion
                .min()
```
It’s  default behavior